corehq/util/view_utils.py

```diff
@@ -68,6 +69,7 @@ def inner(request, *args, **kwargs):
         except BadRequest as e:
             return _get_json_exception_response(400, request, e)
         except Exception as e:
+            notify_exception(request, 'JSON exception response: {}'.format(e))
```

Comment:
> Do we have a policy of considering all bytestrings to be UTF-8 unless we have a good reason to think they aren't?

I think that's been assumed in the past, and I think it's reasonable to try decoding with UTF-8 first to see if it works. Although I don't think it's generally a very good policy because it can result in broken code, and there are some parts of our codebase that use different encodings by default (I think mostly from libraries we didn't write).